developer-docs/recording-format.md

```diff
@@ -0,0 +1,64 @@
++++
+date = "2018-07-19T17:45:00+01:00"
+title = "recording format"
+section_weight = 4
+page_weight = 4
++++
+
+## Recording Format
+
+### Required Files
+
+> Example `info.csv` fil
+```csv
+key,value
+Recording Name,2018_07_19
+Start Date,19.07.2018
+Start Time,14:56:21
+Start Time (System),1532004981.666572
+Start Time (Synced),701730.897108953
+Duration Time,00:00:13
+World Camera Frames,402
+World Camera Resolution,1280x720
+Capture Software Version,1.7.159
+Data Format Version,1.8
+System Info,"User: name, Platform: Linux, ..."
+```
+
+Each recording requires two things:
+1. An `info.csv` file that includes two columns -- `key` and `value`.
+2. At least one video file and its corresponding timestamps. See the [*Video Files*](#video-files) section below for details.
+
+Each row of the `info.csv` file corresponds to a key-value-pair. At least two pairs are required:
```

Comment:
Cosnider`

A minimum requirement of two key, value pairs are required. Example: